pro_tes/utils/db.py

```diff
@@ -106,17 +104,18 @@ def upsert_fields_in_root_object(
         """
         if projection is None:
             projection = {"_id": False}
-        document_unvalidated = self.collection.find_one_and_update(
-            {"worker_id": self.worker_id},
-            {
-                "$set": {
-                    ".".join([root, key]): value
-                    for (key, value) in kwargs.items()
-                }
-            },
-            projection=projection,
-            return_document=ReturnDocument.AFTER,
-        )
+        document_unvalidated = (
+            self.collection.find_one_and_update(
+                {"worker_id": self.worker_id},
+                {
+                    "$set": {
+                        ".".join([root, key]): value
+                        for (key, value) in kwargs.items()
+                    }
+                },
+                projection=projection,
+                return_document=ReturnDocument.AFTER,
+            ))
```

Comment:
Was this formatted by `black`? If not, I think the second closing parenthesis should be on a separate line with less indent.

Reply:
Fixed the indent